core/src/com/mygdx/game/models/BrainstormingPhase.java

```diff
@@ -0,0 +1,62 @@
+package com.mygdx.game.models;
+import java.util.ArrayList;
+
+/**
+ * The BrainstormingPhase keeps track of the brains that have been fired and the wall.
+ *
+ * BRAIN_DAMAGE: How much damage 1 brain does to the wall
+ * Brains: The list of brains that has been sendt to the wall
+ * Wall: The wall
+ *
+ * This class implements the MVC pattern as a model.
+ *
+ * */
+
+public class BrainstormingPhase {
+
+    private final int BRAIN_DAMAGE;
+    private final Wall wall;
+    private ArrayList<Brain> brains = new ArrayList<>();
+
+    /**
+     * The constructor that takes the hitpointmax of the wall, and
+     * how much damage a brain will do
+     * @param wallHitpoints: Walls max HP.
+     * @param BRAIN_DAMAGE: The amount of damage one brain does.
+     */
+    public BrainstormingPhase(int wallHitpoints, int BRAIN_DAMAGE){
+        this.wall = new Wall(wallHitpoints);
+        this.BRAIN_DAMAGE = BRAIN_DAMAGE;
+    }
+
+    public ArrayList<Brain> getBrains() {
+        return brains;
+    }
+
+    public Wall getWall() {
+        return wall;
+    }
+
+    /**
+     * Updates a given brain with an Idea
+     * @param brain: The brain in question.
+     * @param idea: The idea thats going on the brain.
+     * */
+    public void putIdeaOnBrain(Brain brain, Idea idea){
+        brain.addIdea(idea);
+    }
```

Comment:
Depending on how we want to structure the code, we could take advantage of the elimination phase being mostly singleplayer. If we do, it would be a good idea to include a player in the constructor, and create idea objects in this class instead of expecting the round to know. 

Again this depends on what part of the code we decide should do this, so you can ignore this comment if you disagree.

Reply:
I dont disagree, I just wasn't sure how we were gonna sturucture it 😄 So i can change it!